app/components/layer-file/component.js

```diff
@@ -2,10 +2,19 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
     showSelect: false,
+    noFileFound: true,
     didRender() {
         if(!this.get('layer.settings.values.downloadLink')){
             this.get('node.files').then((result)=>{
                 result.objectAt(0).get('files').then((files)=>{
+                    if(files.length === 0){
+                        console.log('no files')
+                        $(".no-files-on-osf").text('No Files found in OSF Project.')
```

Comment:
Let's use native ember methods for this. just show and hide toggle and add text to template.